renku/core/workflow/plan_factory.py

```diff
@@ -79,19 +79,21 @@ def __init__(
         self.no_output_detection = no_output_detection
 
         if not command_line:
-            raise errors.UsageError("Command line can not be empty.")
+            raise errors.UsageError("Command line can not be empty. Please specify a command to execute.")
 
         if not directory:
             directory = os.getcwd()
         self.directory = Path(directory)
         if not self.directory.exists():
-            raise errors.UsageError("Directory must exist.")
+            raise errors.UsageError(
+                "Current directory doesn't exist. Please make sure you are inside an existing directory."
```

Comment:
This message looks a bit weird to me. I'd change to `f"Directory doesn't exist: {self.directory}"

Reply:
This part of the code is really misleading. `directory` gets set to `os.getcwd()` and `working_dir` gets set to `client.repository.path`. And it's not something a user can set.

The error above only happens if someone does `cd somedir && rm -r ../somedir && renku run ...`, I think.